### DISCUSSION TITLE
Long time no see fix this bug. Livewire component nested property not real update value, miss word when typing input using model.live set slug to other form field on updated Lifecycle Hooks

### DISCUSSION THREAD

**Discussion by firer777** [livewire/livewire#8325] [7. Bugs (archived)]
### Livewire version

v3.4

### Laravel version

v11

### Which PHP version are you using?

PHP 8.3

### Repository or Wirebox URL

https://wirebox.app/b/x2180

### Additional information

Blade:

input wire:model.live="form.name" type="text" class="form-control mb-2" tabindex="-1"

Livewire Component:
public PostForm $form;

public function updatedFormName($value)
{
        $this->form->slug = Str::slug($value);
}

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@sinhnv1991 can you create a https://wirebox.app that demonstrates the issue and make sure to make it public? Thanks!

**Reply by firer777** [reply]
https://wirebox.app/b/x2180
Issue when using nested property

**Reply by firer777** [reply]
I think livewire should improve cancel old xhr request and send only lasted request on future will better UX and performance

**Reply by firer777** [reply]
@joshhanley please fix it

**Comment by firer777** [comment]
I updated. Thanks

Vào 3:58, Th 6, 19 thg 4, 2024 Josh Hanley ***@***.***> đã
viết:

**Comment by firer777** [comment]
@joshhanley please fix it
